docs/quickstart/readme.md

```diff
@@ -0,0 +1,310 @@
+
+## Table of Contents
+
+- [Install the Blender Addon](#install-the-blender-addon)
+- [Setup the Bevy side](#setup-the-bevy-side)
+- [Setup the Blender addon for your project](#setup-the-blender-addon-for-your-project)
+- [Create a blueprint](#create-a-blueprint)
+- [Compose the world](#compose-the-world)
+- [Run your game](#run-your-game)
+- [Next Steps](#next-steps)
+
+## Install the Blender Addon
+
+- Download `blenvy.zip` from the [release page](https://github.com/kaosat-dev/Blenvy/releases/tag/blenvy_v0.1.0_pre_alpha)
+- Open Blender
+- Drag and drop `blenvy.zip` into Blender
+  - <details>
+    <summary>This window should show up</summary>
+
+    <img src="img/install.png" width=50%>
+
+    </summary>
+    </details>
+- Leave all settings as is and click on `OK`
+
+Leave the scene open, we will get back to it later.
+
+## Setup the Bevy side
+
+Run the following commands:
+
+```sh
+cargo new my_game
+cd my_game
+cargo add bevy
+cargo add blenvy --git https://github.com/kaosat-dev/Blenvy/ --branch blenvy
+```
+
+This guide will tell you to `cargo run` at multiple points. We expect that you are still in the `my_game` directory when you do so.
+
+Now, replace the contents of `src/main.rs` with the following:
+
+<details>
+<summary>src/main.rs</summary>
+
+```rust
+use bevy::prelude::*;
+use blenvy::*;
+
+fn main() -> AppExit {
+    App::new()
+        .add_plugins((DefaultPlugins, BlenvyPlugin::default()))
+        // We need to register components to make them visible to Blenvy
+        .register_type::<Player>()
+        .add_systems(Startup, setup)
+        .run()
+}
+
+#[derive(Component, Reflect)]
+#[reflect(Component)]
+struct Player {
+    strength: f32,
+    perception: f32,
+    endurance: f32,
+    charisma: f32,
+    intelligence: f32,
+    agility: f32,
+    luck: f32,
+}
+
+fn setup(mut commands: Commands) {
+    commands.spawn((
+        BlueprintInfo::from_path("levels/World.glb"),
+        SpawnBlueprint,
+        HideUntilReady,
+        GameWorldTag,
+    ));
+}
+```
+
+</details>
+
+`Player` is the component that we will add to our entities in Blender. It can contain any fields you want, but you need to remember to call `register_type` with it.
+
+Running this won't work yet because the `levels/World.glb` we reference doesn't exist yet. We will create it in the next step.
+
+## Setup the Blender addon for your project
+
+Create a directory under `my_game` called `art`. Hop back into Blender and save the default scene as `my_game.blend` in the `art` directory we just created. Your file structure should now look like this:
+
+<details>
+<summary>File structure</summary>
+
+<img src="img/art_file_structure.png" width = 50%/>
+</details>
+
+Now, clear the default scene of any objects or collections.
```

Comment:
So this is the only part I am not 100 % on board with: for a Blender tutorial I can understand the reason for clearing the scene & adding a camera & light back, so users know how to do that, but would it not make more sense to just go the "default" route: ie something like
" by default, Blender already adds a camera & a light for you " etc ?
It would be less information for the user to grok for a quickstart imho

Reply:
Ha, I had it that way first, but then I changed it for the following reasons:

First, there is a default collection that I would like to remove, because otherwise we get into the question of "does it matter if I add my collection instance into or outside of the collection?"  
We both know it does not matter, but from screenshots it will quickly look that way, requiring us to specify that this collection is "just for show" and we should ignore it. Or we delete it, forcing us to explain the Blender quirk of "You see, when you delete the collection, you actually don't delete the contents; they just kinda spill over"  
The user will certainly run into this themselves at some point, but I'd like to keep the quickstart clean of that. The simplest way is to just start from a completely blank slate.

The second reason is that for me personally, when I first used the Blender workflow, it kinda felt like magic how I got a camera and a light in there because I completely forgot about the default cam + light, as they feel like "background information", if that makes sense. Only when I added a camera and light myself did I get an intuition for "Oh, the workflow actually reads these!"
Plus, you get to see that not only spotlights are supported, which is neat.

Reply:
Very sound reasoning! (Also to this day I still do not understand the logic of Blende for adding that extra collection,it is a relatively recent addition too, I think from 4.1...)
Okay so how about this : instead of removing/ changing what you already wrote, just adding a small "info" something like " Blender creates a camera and a light and an additional collection by default, you can also use those , but for this quick start we go into a bit more detail for clarity" ?

Reply:
Does this sound good? [`0e086c3` (#213)](https://github.com/kaosat-dev/Blenvy/pull/213/commits/0e086c342c0a868779852c29e28cdebd6a1dbc5f)